src/common/hooks/useResolveDayJSLocale.ts

```diff
@@ -0,0 +1,303 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export function useResolveDayJSLocale() {
```

Comment:
@beganovich, this is the only reliable way I found to import the correct locale from dayjs/locale after a lot of research. The problem is that if we ignore Vite for dynamic imports, we'll run into issues with webpack not allowing it. The issue is that we need to put variables into the path from which we import something. (That is actually something is not allowed by webpack)

Here's a reference for this:

https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import

I know that hard-coding isn't ideal, and I've talked to David about it, but we must not change this even if we add more languages to our app because it contains all available languages from dayjs. So, our app can't have more languages than what dayjs offers. The `fetch` method won't help here also. I'm open to your suggestions, but I haven't found a better solution for this issue. Let me know what you think.

Reply:
This looks fine. I am not sure what it has to do with Webpack, we don't use it. Vite uses Rollup to bundle assets, but ok.

Nonetheless, it's an ok approach :+1: